spec/openapi.yaml

```diff
@@ -1138,6 +1157,40 @@ components:
                 description: Path to package
                 example: "jo/packages"
 
+    Transfer:
+      type: object
+      properties:
+        type:
+          type: string
+          enum: [ NO_TRANSFER, FROM_WEB, FROM_CONNECT, FROM_PRINTER, FROM_SLICER, FROM_CLIENT, TO_CONNECT, TO_CLIENT ]
+          example: FROM_WEB
+        path:
+          type: string
+          example: "/PrusaLink gcodes/Spice_Harvester_0.3mm_PLA_MK3S_12m.gcode"
```

Comment:
LFN or SFN? (I assume SFN).

Reply:
@ondratu I am not sure about this?

Reply:
It is destination, so at this moment, which have to be created after full download. So SFN is not available at this moment.

Question is, if it should be path and display_name just like in FileInfo ?

Reply:
Well, it can be a mix ‒ path being SFN, the last segment LFN. So, is there a _requirement_ for it to be human readable, or is this „machine identification of resource“?